### DISCUSSION TITLE
Pyvista Voxelization Voxels size

### DISCUSSION THREAD

**Discussion by SofianeHamlaoui** [pyvista/pyvista#4578] [Q&A] [answered]
Hello, 

I know we can play with density while voxelizing a mesh, but is it possible to make it with custom Voxel sizes ( x,y,z ) ? 

![image](https://github.com/pyvista/pyvista/assets/16967174/669d7a28-442b-4e2f-b726-64630c65efb2)

**Comment by banesullivan** [comment]
This has been requested a few times. It's an easy fix if you're open to contributing it. Take a look at the `voxelize()` method and likely just swap the density for your sizes or build the meshgrid however you like here:

https://github.com/pyvista/pyvista/blob/392d29ea4398c7292275300b44b75bcdbbed1c2e/pyvista/core/utilities/features.py#L78-L82

**Reply by banesullivan** [reply, marked as answer]
FYI https://github.com/pyvista/pyvista/issues/4312
